libs/dh/charges/feature-charges/src/components/charge.component.ts

```diff
@@ -49,26 +54,61 @@ import { TranslocoDirective } from '@jsverse/transloco';
       overflow: auto;
     }
   `,
-  imports: [TranslocoDirective, VaterStackComponent, VaterSpacerComponent, WATT_LINK_TABS],
+  imports: [
+    TranslocoPipe,
+    TranslocoDirective,
+    VaterStackComponent,
+    VaterSpacerComponent,
+    WATT_LINK_TABS,
+    DhEmDashFallbackPipe,
+    DhChargeStatusComponent,
+    DhChargeActionsComponent,
+  ],
   template: `<div class="page-grid">
     <div class="page-header" vater-stack direction="row" gap="m" wrap align="end">
       <div *transloco="let t; prefix: 'charges.charge'">
-        <h2 vater-stack direction="row" gap="m" class="watt-space-stack-s">headline</h2>
+        <h2 vater-stack direction="row" gap="m" class="watt-space-stack-s">
+          {{ charge()?.chargeId | dhEmDashFallback }} {{ charge()?.chargeName | dhEmDashFallback }}
+          @let status = charge()?.status;
+
+          @if (status) {
+            <dh-charge-status [status]="status" />
+          }
+        </h2>
 
         <vater-stack direction="row" gap="ml">
-          <span class="watt-text-s"> info </span>
+          <span class="watt-text-s">
+            <span class="watt-label watt-space-inline-xs">{{ t('type') }}</span>
+
+            {{ 'charges.chargeTypes.' + charge()?.chargeType | transloco }}
```

Comment:
nitpick: all places where a property can be `undefined` (e.g. `charge()?.chargeType`) will result in warnings in the console about missing translations (until the value is received). In those cases we've used `if`s to guard against it.